pkg/kube/inject/webhook.go

```diff
@@ -1110,13 +1111,15 @@ func (wh *Webhook) inject(ar *kube.AdmissionReview, path string) *kube.Admission
 	if pod.ObjectMeta.Namespace == "" {
 		pod.ObjectMeta.Namespace = req.Namespace
 	}
-	log.Infof("Sidecar injection request for %v/%v", req.Namespace, podName)
+
+	log = log.WithLabels("ns", req.Namespace, "pod", pod.Namespace+"/"+podName)
+	log.Infof("Process sidecar injection request")
```

Comment:
We don't need this info logs I think, will be spammy?

Reply:
This line is modified based on L1113 on the left (see left, the old code), the old:
```
log.Infof("Sidecar injection request for %v/%v", req.Namespace, podName)
```
I think it should be fine to keep it, but no strong demand.

Reply:
ah my bad thanks. Let's keep it